### DISCUSSION TITLE
How do you add mods?

### DISCUSSION THREAD

**Discussion by jsDhillon8** [AlexProgrammerDE/balena-minecraft-server#127] [Q&A]
Hey Alex, I was just wondering if you know how to add mods. I wish to add the proximity chat mod. let me know. thx

**Comment by viedok** [comment]
So... I think you are talking about a plugin. The difference is that a mod runs on the client (your game) and a plugin is a sort of mod for the server itself.
So to add plugins you must download them from the [spigot website](https://www.spigotmc.org/resources/categories/spigot.4/) where all the compatible plugins are (because Paper, the server type you are using, is based on spigot).
I don't know what type of proximity chat you exactly want to use but I have tried some and if you want "true" proximity chat, you'll have to use an other thing than discord like mumble.
If you still want to use discord, i'll recommend using skoice, the voices don't fade away like on a "true" proximity chat mod but it gets the job done and moves users in different channels regarding their location on the server.
I tried to be as clear as possible but if you want more info feel free to contact me on discord :) (taediumvitae#6515)

**Comment by viedok** [comment]
I'm happy to announce that I've found a solution ! It's by using a [fabric server](https://fabricmc.net/use/) instead of a typical one and the [Simple Voice Chat Mod](https://www.curseforge.com/minecraft/mc-mods/simple-voice-chat). With all of this, you'll be able to use a REAL proximity chat !
Feel free to reply if you wonder how to use it. :)
